docs/Webpack.md

```diff
@@ -127,7 +143,7 @@ Remember to include the default `babel-jest` transformer explicitly, if you wish
 
 ### Configuring Jest to find our files
 
-Now that Jest knows how to process our files, we need to tell it how to _find_ them. For webpack's `modulesDirectories`, and `extensions` options there are direct analogs in Jest's `moduleDirectories` and `moduleFileExtensions` options.
+Now that Jest knows how to process our files, we need to tell it how to _find_ them. For Webpack's `modules` and `extensions` options there are direct analogs in Jest's `moduleDirectories` and `moduleFileExtensions` options.
 
 ```json title="package.json"
```

Comment:
Is there a reason to configure the things in package.json and not giving the code examples with the separated jest.config.(ts/js) file?